lib/Service/ContactsIntegration.php

```diff
@@ -106,10 +106,12 @@ public function getMatchingRecipient(string $term): array {
 	 * @return false|null|string
 	 */
 	public function getPhoto(string $email) {
-		$result = $this->contactsManager->search($email, ['EMAIL']);
+		$result = $this->contactsManager->search($email, ['EMAIL'], ['escape_like_param' => false]);
```

Comment:
Could you submit this change as a separate PR? I would like to have a closer look at this. The other change can go in earlier :)